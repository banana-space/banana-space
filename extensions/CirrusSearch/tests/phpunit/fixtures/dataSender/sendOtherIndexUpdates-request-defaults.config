{
  "config": {
      "CirrusSearchIndexBaseName": "myfixturewiki",
      "CirrusSearchUpdateShardTimeout": "1ms"
  },
  "indexName": "myfixturetargetindex",
  "localSite": "myfixturesourceindex",
  "batchSize": 2,
  "actions": [
    {
      "docId": 1,
      "dbKey": "Foo",
      "ns": 0
    },
    {
      "docId": 2,
      "dbKey": "Bar_baz",
      "ns": 3
    },
    {
      "docId": 3,
      "dbKey": "Not_Inspired",
      "ns": 2
    }
  ]
}
