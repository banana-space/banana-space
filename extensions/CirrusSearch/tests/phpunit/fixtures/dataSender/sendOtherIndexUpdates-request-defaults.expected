{
    "path": "_bulk",
    "method": "POST",
    "data": [
        [
            {
                "update": {
                    "_index": "myfixturetargetindex",
                    "_type": "page",
                    "_id": 1
                }
            },
            {
                "script": {
                    "source": "super_detect_noop",
                    "params": {
                        "source": {
                            "local_sites_with_dupe": {
                                "remove": "myfixturesourceindex"
                            }
                        },
                        "handlers": {
                            "local_sites_with_dupe": "set"
                        },
                        "_id": 1,
                        "_type": "page",
                        "_index": "myfixturetargetindex"
                    },
                    "lang": "super_detect_noop"
                }
            },
            {
                "update": {
                    "_index": "myfixturetargetindex",
                    "_type": "page",
                    "_id": 2
                }
            },
            {
                "script": {
                    "source": "super_detect_noop",
                    "params": {
                        "source": {
                            "local_sites_with_dupe": {
                                "remove": "myfixturesourceindex"
                            }
                        },
                        "handlers": {
                            "local_sites_with_dupe": "set"
                        },
                        "_id": 2,
                        "_type": "page",
                        "_index": "myfixturetargetindex"
                    },
                    "lang": "super_detect_noop"
                }
            }
        ],
        [
            {
                "update": {
                    "_index": "myfixturetargetindex",
                    "_type": "page",
                    "_id": 3
                }
            },
            {
                "script": {
                    "source": "super_detect_noop",
                    "params": {
                        "source": {
                            "local_sites_with_dupe": {
                                "remove": "myfixturesourceindex"
                            }
                        },
                        "handlers": {
                            "local_sites_with_dupe": "set"
                        },
                        "_id": 3,
                        "_type": "page",
                        "_index": "myfixturetargetindex"
                    },
                    "lang": "super_detect_noop"
                }
            }
        ]
    ],
    "params": [],
    "contentType": "application\/x-ndjson"
}